common/djangoapps/enrollment/tests/test_api.py

```diff
@@ -230,3 +230,52 @@ def test_caching(self):
         # The data matches
         self.assertEqual(len(details['course_modes']), 3)
         self.assertEqual(details, cached_details)
+
+    def test_update_enrollment_expired_mode_with_error(self):
+        """ Verify that if verified mode is expired then enrollment cannot be updated if
+        include_expired flag is false."""
```

Comment:
Verify that if verified mode is expired and include expire flag is false then enrollment cannot be updated.